tests/integration/test_bundler.py

```diff
@@ -13,7 +13,7 @@
     [
         pytest.param(
             utils.TestParameters(
-                ref="bundler/missing-gemfile",
```

Comment:
This commit should be squashed into the one consolidating everything into a single repo.